### ISSUE TITLE
This long long is obviously not long enough

### ISSUE THREAD

**Issue by Jackson-Tinsley** [PhiYerion/Intermediary-Refueling-Analysis#6]
i tried measuring the longth and it wasnt long long

**Comment by Jackson-Tinsley** [comment]
wow this is great code, i would hire this guy

**Comment by Jackson-Tinsley** [comment]
this code saved my relationship of 20 years!!!! cant recommend enough

**Comment by Jackson-Tinsley** [comment]
this code got me laid, thanks a ton!

**Comment by PhiYerion** [comment]
You are right. I am implementing mpfr for up to 256 bit precision, specifically for the delta-V calculation involving log.

**Comment by PhiYerion** [comment]
You can check out the branch for the issue if you would like to contribute to it's integration.
